cmux-tui/bindings/verify_github_environment_policy.py

```
@@ -0,0 +1,216 @@
+    )
+
+
+def _fetch_environment(
+    repository: str,
+    environment: str,
+    token: str,
+    *,
+    api_url: str = GITHUB_API_URL,
+) -> dict[str, Any]:
+    if not token:
+        raise EnvironmentPolicyError("GitHub API token is missing")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)